<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spline interpolation 32fc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #fff;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="input_LED">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 15px;
}
QLineEdit:focus {
  border: 1px solid #d0e3ff;
}
QLineEdit::placeholder {
  color: #767e89;
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">QLabel {
  color: #464d55;
  font-weight: 600;
}</string>
         </property>
         <property name="text">
          <string>Input</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="output_LED">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
  border-radius: 8px;
  border: 1px solid #e0e4e7;
  padding: 5px 15px;
}
QLineEdit:focus {
  border: 1px solid #d0e3ff;
}
QLineEdit::placeholder {
  color: #767e89;
}</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">QLabel {
  color: #464d55;
  font-weight: 600;
}</string>
         </property>
         <property name="text">
          <string>Output</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="input_PBN">
         <property name="styleSheet">
          <string>QPushButton {
  background-color: #0d6efd;
  color: #fff;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid #0d6efd;
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}
QPushButton:hover,
QPushButton:focus {
  background-color: #0b5ed7;
  border: 3px solid #9ac3fe;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/icons8-opened-folder-94.png</normaloff>:/res/icons8-opened-folder-94.png</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="output_PBN">
         <property name="styleSheet">
          <string>QPushButton {
  background-color: #0d6efd;
  color: #fff;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid #0d6efd;
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}
QPushButton:hover,
QPushButton:focus {
  background-color: #0b5ed7;
  border: 3px solid #9ac3fe;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/res/icons8-opened-folder-94.png</normaloff>:/res/icons8-opened-folder-94.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">QLabel {
  color: #464d55;
  font-weight: 600;
}</string>
         </property>
         <property name="text">
          <string>Sample rate new, Hz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="FdNew_SBX">
         <property name="maximum">
          <number>1000000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">QLabel {
  color: #464d55;
  font-weight: 600;
}</string>
         </property>
         <property name="text">
          <string>Sample rate old, Hz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSlider" name="FdNew_SL">
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="FdOld_SBX">
         <property name="sizeIncrement">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSlider" name="FdOld_SL">
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="start_PBN">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
  background-color: #0d6efd;
  color: #fff;
  font-weight: 600;
  border-radius: 8px;
  border: 1px solid #0d6efd;
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}
QPushButton:hover,
QPushButton:focus {
  background-color: #0b5ed7;
  border: 3px solid #9ac3fe;
}</string>
           </property>
           <property name="text">
            <string>start</string>
           </property>
           <property name="iconSize">
            <size>
             <width>38</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="info_PBR">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">
QProgressBar::chunk
{

border: 1px solid #0d6efd;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:0,stop:0 #01FAFF,stop:1  #26B4FF);
}
QProgressBar#progressBar
{

height:22px;
Text-align: center; / * Text location * /
font-size:14px;
color:white;
border-radius:11px;
background: #1D5573 ;
}
</string>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="log_PTE">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPlainTextEdit{
border-radius: 8px;
border: 1px solid #e0e4e7;
padding: 5px 15px;
}</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>FdOld_SL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FdOld_SBX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FdNew_SL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FdNew_SBX</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FdOld_SBX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FdOld_SL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FdNew_SBX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FdNew_SL</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
